packages/environment-check/package.json

```diff
@@ -0,0 +1,44 @@
+{
+    "name": "@sap-ux/environment-check",
+    "version": "0.0.1",
+    "description": "SAP Fiori environment check",
+    "license": "Apache-2.0",
+    "main": "dist/index.js",
+    "private": false,
+    "scripts": {
+        "envcheck": "./bin/envcheck",
+        "build": "rimraf dist && tsc",
+        "clean": "rimraf dist test/test-output coverage",
+        "watch": "tsc --watch",
+        "lint": "eslint . --ext .ts",
+        "lint:fix": "eslint . --ext .ts --fix",
+        "test": "jest --ci --forceExit --detectOpenHandles --colors",
+        "test-u": "jest --ci --forceExit --detectOpenHandles --colors -u",
+        "link": "pnpm link --global",
+        "unlink": "pnpm unlink --global"
+    },
+    "dependencies": {
+        "@sap/bas-sdk": "^2.1.0",
+        "@sap-ux/btp-utils": "workspace:*",
+        "@sap-ux/logger": "workspace:*",
+        "@types/archiver": "5.1.1",
+        "archiver": "5.3.0",
+        "axios": "0.26.0",
```

Comment:
Please make sure that you use the same `axios` version as in the other modules in the monorepo.

Reply:
Thanks